types/src/aggregated_signature.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_crypto::bls12381;
+use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use std::collections::HashMap;
+
+use move_deps::move_core_types::account_address::AccountAddress;
+use serde::{Deserialize, Serialize};
+
+/// This struct represents the aggregated BLS signature representation that contains an aggregated
+/// BLS signature and a bit mask representing the set of validators participating in the signing
+/// process
+#[derive(
+    Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize, CryptoHasher, BCSCryptoHash,
+)]
+pub struct AggregatedSignature {
```

Comment:
I can already imagine @alinush chime in here, this is a MultiSignature (on the same message) instead of a AggregatedSignature

Reply:
Oh dear God! :) Are the aggregated signatures on the same message? If so, please rename to multisignature.

Reply:
@alinush - I wanted to have some fun by creating this confusion 😁

Reply:
LOL, should I review it like this or wait for you to fix it?

Reply:
Let's wait - this is not ready for review yet. Will publish this for review once I fix the tests.